one-ui/src/test/java/com/marklogic/hub/oneui/AuthTest.java

```diff
@@ -46,14 +48,67 @@ void loginWithInvalidCredentials() throws Exception {
     }
 
     @Test
-    void loginWithValidCredentialsAndLogout() throws Exception {
+    void loginWithValidAdminAndLogout() throws Exception {
         String payload = getLoginPayload("admin", "admin");
         final MockHttpSession session[] = new MockHttpSession[1];
         // Login
         mockMvc
             .perform(post(LOGIN_URL).contentType(MediaType.APPLICATION_JSON).content(payload))
-            .andDo(
-                result -> session[0] = (MockHttpSession) result.getRequest().getSession())
+            .andDo (
+                result -> {
+                    session[0] = (MockHttpSession) result.getRequest().getSession();
+                    assertTrue(result.getResponse().getContentAsString().contains("\"hasManagePrivileges\":true"));
+                })
+            .andExpect(status().isOk());
+
+        assertFalse(session[0].isInvalid());
+
+        // Logout
+        mockMvc.perform(get(LOGOUT_URL).session(session[0]))
+            .andExpect(status().isOk());
+
+        assertTrue(session[0].isInvalid());
+    }
+
+    @Test
+    void loginWithDataHubManagerAndLogout() throws Exception {
+        String payload = getLoginPayload("data-hub-environment-manager-user", "data-hub-environment-manager-user");
+        final MockHttpSession session[] = new MockHttpSession[1];
+        // Login
+        mockMvc
+            .perform(post(LOGIN_URL).contentType(MediaType.APPLICATION_JSON).content(payload))
+            .andDo (
+                result -> {
+                    session[0] = (MockHttpSession) result.getRequest().getSession();
+                    String strResponse = result.getResponse().getContentAsString();
+                    ObjectMapper mapper = new ObjectMapper();
+                    JsonNode jsonResonse = mapper.readTree(strResponse);
+                    assertTrue(jsonResonse.get("roles").isArray());
+                    assertTrue(strResponse.contains("\"hasManagePrivileges\":true"));
```

Comment:
@ryanjdew we are still at the early stages of 5.3 so maybe we can change this now. Is hasManagePrivileges telling us if the user has manage-admin role? If so, then shouldnt we name it that way. I am saying this because data-hub-operator has manage privilege, will this flag then be true.

Reply:
It isn't explicitly depending on whether we have access to the Data Service endpoint it is really either don't get forbidden from the manage REST app server or if the Data Service endpoint is available it was checking the role, but now should check "canInstallDataHub".